R/align_lineage.R

```diff
@@ -41,22 +41,25 @@
 #' must contain 'Cluster' column, which is added by seqCluster() function, and 'Sequence.germline'
 #' column, which is added by repGermline() function.
 #'
+#' @param .verbose_output If TRUE, all output dataframe columns will be included (see documentation about this
+#' function return), and unaligned clusters will be included in the output. Setting this to TRUE significantly
+#' increases memory usage.
```

Comment:
And if `FALSE`?

Reply:
Details are below, in the description of @return: notices for columns when they are included only when .verbose_output=TRUE